_posts/2015-12-04-keep-yourself-sane-with-motd-ascii-art.md

```diff
@@ -0,0 +1,229 @@
+
+A few days later my passion for logging into the wrong server was greeted by:
+
+<pre style="font-family:'Lucida Console', 'Courier New', 'Courier', monospace; font-size:9px; background-color:#000; color:#fff; line-height:1;">
+                                                     ___,------, 
+             _,--.---.                         __,--'         / 
+           ,' _,'_`._ \                    _,-'           ___,| 
+          ;--'       `^-.                ,'        __,---'   || 
+        ,'               \             ,'      _,-'          || 
+       /                  \         _,'     ,-'              || 
+      :                    .      ,'     _,'                 |: 
+      |                    :     `.    ,'                    |: 
+      |           _,-      |       `-,'                      :: 
+     ,'____ ,  ,-'  `.   , |,         `.                     : \ 
+     ,'    `-,'       ) / \/ \          \                     : : 
+     |      _\   o _,-'    '-.           `.                    \ \ 
+      `o_,-'  `-,-' ____   ,` )-.______,'  `.                   : : 
+       \-\    _,---'    `-. -'.\  `.  /     `.                  \  \ 
+        / `--'             `.   \   \:        \                  \,.\ 
+       (              ____,  \  |    \\        \                 :\ \\ 
+        )         _,-'    `   | |    | \        \                 \\_\\ 
+       /      _,-'            | |   ,'-`._      _\                 \,' 
+       `-----' |`-.           ;/   (__ ,' `-. _;-'`\           _,--' 
+     ,'        |   `._     _,' \-._/  Y    ,-'      \      _,-' 
+    /        _ |      `---'    :,-|   |    `     _,-'\_,--'   \ 
+   :          `|       \`-._   /  |   '     `.,-' `._`         \ 
+   |           _\_    _,\/ _,-'|                     `-._       \ 
+   :   ,-         `.-'_,--'    \                         `       \ 
+   | ,'           ,--'      _,--\           _,                    : 
+    )         .    \___,---'   ) `-.____,--'                      | 
+   _\    .     `    ||        :            \                      ; 
+ ,'  \    `.    )--' ;        |             `-.                  / 
+|     \     ;--^._,-'         |                `-._            _/_\ 
+\    ,'`---'                  |                    `--._____,-'_'  \ 
+ \_,'                         `._                          _,-'     ` 
+                            ,-'  `---.___           __,---' 
+                          ,'             `---------' 
+                        ,' 
+</pre> 
+
+<blockquote>
+"This is not the server I'm looking for. Thanks Homer"
+</blockquote>
+
+Updating the MOTD file to include a visual queue has been part of my server setups ever since. It's small tweak, but has prevented serious mistakes. (That and better sleeping habits)
+
+<h2>How to Customize MOTD (message of the day)</h2>
```

Comment:
I'd recommend using markdown syntax instead of straight HTML.

For example the equivalent of `<h2>` in markdown would be `##`

So instead of: `<h2>Title goes here</h2>`
You should use: `## Title goes here` (no closing tags or anything)